_src/_sass/_buttons.scss

```diff
@@ -27,17 +27,10 @@
 }
 
 #award {
-  background-image: url("/assets/img/mobile-award.png");
-  background-size: 167px 47px;
-  width: 167px;
-  height: 47px;
-  margin-bottom: 2em;
-
   @media (min-width: 64em) {
-    background-image: url("/assets/img/award.png");
-    background-size: 170px 100px;
-    width: 170px;
-    height: 100px;
+    background-image: url("/assets/img/apache-foundation-logo.png");
+    background-repeat: no-repeat;
+    background-size: contain;
```

Comment:
This whole section is why the VLDB image has been accidentally replaced with the Apache top-level image.  Can you go back to the original `award` style id here, and then create a new style id for the top-level project announcement?  Alternatively, it might make sense to pull out a common style class, and then an id for each with the rest of the specifics.